incubator/kube-ops-view/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 | trimSuffix "-" -}}
```

Comment:
Could you change this to truncate to 63 chars as per https://github.com/kubernetes/charts/issues/233? Thanks!